packages/tools-workspaces/src/lerna.ts

```diff
@@ -19,11 +19,13 @@ function filterSentinels(): string[] {
 export async function findWorkspacePackages(
   configFile: string
 ): Promise<string[]> {
-  const { packages, useWorkspaces } = await readJSON(configFile);
-  if (!useWorkspaces) {
+  const { packages } = await readJSON(configFile);
+  // respect the Lerna `packages` property
+  if (packages) {
```

Comment:
Would it make sense to do an explicit check here?

```suggestion
  if (packages && useWorkspaces !== true) {
```

Reply:
I honestly don't know what's the expected pre-7 Lerna behaviour here. What should happen if user expicitly sets `useWorkspaces` to false, but still provides `packages` list. I bet people at Lerna didnt know too so they just decided to deprecate `useWorkspaces`.

Could you please remind what's your preferred style is? Is it amends or do i just slap commits on top?

Reply:
It looked like they had a use case for it a long time ago. I think it makes sense to keep it for now.

Can you please push additional commits? That way we have the history of changes here. We will squash before rebasing and merging with main.